### ISSUE TITLE
Remove notifications, that can't be delivered

### ISSUE THREAD

**Issue by tantra35** [moira-alert/notifier#2]
After mistake in config file we send notification throw email transport(agent), but this agent doesn't exists (its renamed to mail), so notifier constantly write in it log

```
2016-01-14 18:28:29     DEBUG   Scheduled notification for contact email:r****@****m trigger  at 2016/01/14 18:29:29 (1452785369)
2016-01-14 18:28:29     DEBUG   Scheduled notification for contact email:r****@****m trigger  at 2016/01/14 18:29:29 (1452785369)
2016-01-14 18:29:29     WARNING Can't send message after 896 try: Unknown contact type [package of 2 notifications to r****@****m]. Retry again 

```

And forever

**Comment by beevee** [comment]
It's not forever, it's for 1 day :)

@gmlexx maybe we should make this ttl configurable?

**Comment by gmlexx** [comment]
Or maybe we need some API to delete them

**Comment by gmlexx** [comment]
This feature is available in v1.1.14 of worker (API) and v1.0.19 of web
Open /#/notifications/ page and remove failed items.
